package.json

```diff
@@ -16,7 +16,8 @@
     "strdup": "0.0.0",
     "Constellation/console-colors.c": "1.0.1",
     "littlstar/asprintf.c": "0.0.3",
-    "logger": "0.0.0"
+    "logger": "0.0.0",
+    "kgabis/parson": "master"
```

Comment:
@darthtrevino i've forked the original `parson` repo and reverted the `\/` nonsense.  can you update to use [`clibs/parson@1.0.0`](https://github.com/clibs/parson/tree/1.0.0)?